### ISSUE TITLE
[SUGGESTION] Add a pop up / message saying the player needs to sway while brushing

### ISSUE THREAD

**Issue by plantiac** [capsey/archeology-mod#66]
\*me\*: blocks get destroyed and become frustrated for the mod not working (even though it is but I don't know). 
\*also me\*: checks out mod description on Curseforge and just finds out I have to sway the cursor to prevent block breaking

What I'm trying to say is either highlight that info on Curseforge or even better (especially for players who don't check out mods on Curseforge and don't know of Minecraft Live 2020) send a message in-game as soon as the player starts brushing, letting them know to move the cursor.

**Comment by capsey** [comment]
Related #60

**Comment by capsey** [comment]
Do you suggest to implement it like the "There are monsters nearby" and "Bed too far" messages? I also think I can make so that the message stops appearing after 1 or 3 successful excavation to not be annoying.

**Comment by plantiac** [comment]
Yes like these messages. Letting it stop appearing is also great and maybe let it appear again after 3 failed attempts (in case player forgets or so)

**Comment by capsey** [comment]
Related #57
